typescript/organisations.d.ts

```diff
@@ -30,6 +30,7 @@ export type NewOrganisation = {
   cdnHostingAwsAccountId: string
   formStoreFormIds?: number[]
   tierOverrides?: Partial<NewTier['tierData']>
+  attachmentLinkExpiryDays?: number
```

Comment:
The other "..Days" types are `?: number | null` should this be the same?

Reply:
Don't think so, its just left over from flow